source/tyr/tyr/default_settings.py

```diff
@@ -14,13 +14,13 @@
 # URI for postgresql
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 # http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-SQLALCHEMY_DATABASE_URI = 'postgresql://navitia:navitia@localhost/jormungandr'
+SQLALCHEMY_DATABASE_URI = 'postgresql://navitia:navitia@10.0.3.14/jormungandr'
 
 
 # URI for cities database
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 # http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-CITIES_DATABASE_URI = os.getenv('TYR_CITIES_DATABASE_URI', 'postgresql://navitia:navitia@localhost/cities')
+CITIES_DATABASE_URI = os.getenv('TYR_CITIES_DATABASE_URI', 'postgresql://navitia:navitia@10.0.3.14/cities')
```

Comment:
Oups, :speak_no_evil: the `default_settings.py` has to be the same

Reply:
:+1: